specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-30/disk.json

```diff
@@ -1634,6 +1634,10 @@
         "diskAccessId": {
           "type": "string",
           "description": "ARM id of the DiskAccess resource for using private endpoints on disks."
+        },
+        "tier": {
+          "type": "string",
+          "description": "Customer visible tier on the disk which indicates the tier to be set on the blob."
```

Comment:
Performance tier of the disk (e.g, P4, S10) as described here: https://azure.microsoft.com/en-us/pricing/details/managed-disks/. Does not apply to Ultra disks